src/main/java/vct/col/util/FindBinders.scala

```diff
@@ -0,0 +1,12 @@
+package vct.col.util
+
+import vct.col.ast.expr.BindingExpression
+import vct.col.ast.stmt.decl.ProgramUnit
+import vct.col.ast.util.{ASTVisitor, RecursiveVisitor}
+import hre.lang.System._
+
+case class FindBinders(override val source: ProgramUnit) extends RecursiveVisitor[Unit](source) {
```

Comment:
This is for debugging right? Do you think it will be useful in the future? If so, maybe add a brief documentation line stating it's there for debugging purposes (in case we forget)

Reply:
deleted